internal/config/validate.go

```diff
@@ -0,0 +1,65 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+package config
+
+import "fmt"
+
+func (nodeConfig *NodeConfig) Validate() {
+	if nodeConfig.cartesiExperimentalSunodoValidatorEnabled != nil &&
+		*nodeConfig.cartesiExperimentalSunodoValidatorEnabled {
+		if nodeConfig.cartesiExperimentalSunodoValidatorRedisEndpoint == nil {
+			panic("missing CartesiExperimentalSunodoValidatorRedisEndpoint config")
```

Comment:
I think it's better to inform with the format the configuration is defined:

```suggestion
			panic("CARTESI_EXPERIMENTAL_SUNODO_VALIDATOR_REDIS_ENDPOINT environment variable not defined")
```